controllers/platform/platformutils.go

```diff
@@ -142,3 +144,12 @@ func GetRegistryAddress(ctx context.Context, c client.Client) (*string, error) {
 	}
 	return nil, nil
 }
+
+func GetCustomizedDockerfile(dockerfile string, platform operatorapi.SonataFlowPlatform) string {
+	if platform.Spec.BuildPlatform.BaseImage != "" {
+		re := regexp.MustCompile(`FROM (.*) AS builder`)
```

Comment:
You can move this to an init var instead to avoid recompiling it all the time. I did this in another feature that I won't recall now.

Reply:
Optionally, make this string a const

Reply:
Thank you for the suggestion. It cannot be a cost but I made it an init var like you also did in some other cases.
It will be compiled only one time.

Reply:
The const is the string regexp...